src/constants/networks.ts

```diff
@@ -15,7 +15,7 @@ if (typeof INFURA_KEY === 'undefined') {
  *
  * These "Safe" URLs are listed first, followed by other fallback URLs, which are taken from chainlist.org.
  */
-export const FALLBACK_URLS: { [key in SupportedChainId]: string[] } = {
+export const FALLBACK_URLS = {
```

Comment:
why not keep the typing here?

Reply:
it keeps the same resulting type via inference, but it doesn't require exhaustive enumeration of the enum in the declaration

Reply:
🤔 dont understand dis

Reply:
updated phrasing, sorry!

Reply:
in the widget we use this type: `Record<SupportedChainId, string[]>`

Reply:
wait so you should really have a separate `SupportedChainId` just locally to interface and then this map should exhaustively enumerate through that